dauphin/exporters.py

```diff
@@ -17,14 +17,14 @@ def __init__(self, file, *args, **kwargs):
 
     def start_exporting(self):
         self.xg.startDocument()
-        self.xg.startElement(self.rss_element, {'version':'2.0','xmlns:atom':'http://www.w3.org/2005/Atom'})
+        self.xg.startElement(self.rss_element, { 'version':'2.0', 'xmlns:atom':'http://www.w3.org/2005/Atom' })
         self.xg.startElement(self.channel_element, {})
         # TODO: export as self-closing tag + add attr.
         #self._export_xml_field('atom:link', self.channel_atom_link)
         self._export_xml_field('title', self.channel_title)
         self._export_xml_field('link', self.channel_link)
         self._export_xml_field('description', self.channel_description)
-        self._export_xml_field('pubDate', time.strftime("%a, %e %b %Y %H:%M:%S %z", time.localtime()))
+        self._export_xml_field('pubDate', time.strftime("%Y-%m-%dT%H:%m:%S%z", time.localtime()))
```

Comment:
Oui, mais comme dans le top tu as mis que c’était au format atom, et que tu as rajouté des tags atom, j’ai pensé qu’on était en atom et pas en rss2.
Les formats de date atom sont en rfc3339 qui est un subset de iso8601 
https://en.wikipedia.org/wiki/Atom_(standard)#Example_of_an_Atom_1.0_feed <https://en.wikipedia.org/wiki/Atom_(standard)#Example_of_an_Atom_1.0_feed>
Et non le foutu validateur du w3c checke pour des formats de date rss2.0.

> Le 26 juin 2016 à 20:40, Mazamazine <notifications@github.com> a écrit :
> 
> In dauphin/exporters.py <https://github.com/mobula/scrapy_dauphin/pull/4#discussion_r68513648>:
> 
> >          self.xg.startElement(self.channel_element, {})
> >          # TODO: export as self-closing tag + add attr.
> >          #self._export_xml_field('atom:link', self.channel_atom_link)
> >          self._export_xml_field('title', self.channel_title)
> >          self._export_xml_field('link', self.channel_link)
> >          self._export_xml_field('description', self.channel_description)
> > -        self._export_xml_field('pubDate', time.strftime("%a, %e %b %Y %H:%M:%S %z", time.localtime()))
> > +        self._export_xml_field('pubDate', time.strftime("%Y-%m-%dT%H:%m:%S%z", time.localtime()))
> J'avais changé ta ligne pour mettre au format RFC 822 pour suivre les RSS 2.0 specifications:
> 
> pubDate The publication date for the content in the channel. For example, the New York Times publishes on a daily basis, the publication date flips once every 24 hours. That's when the pubDate of the channel changes. All date-times in RSS conform to the Date and Time Specification of RFC 822, with the exception that the year may be expressed with two characters or four characters (four preferred).
> 
> https://validator.w3.org/feed/docs/rss2.html <https://validator.w3.org/feed/docs/rss2.html>
> Pourquoi ISO8601?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/mobula/scrapy_dauphin/pull/4/files/2d7bddff716ea3a2d95b93096fc15282cef72a3c#r68513648>, or mute the thread <https://github.com/notifications/unsubscribe/ACEbu6gGav5f5c0G6jwZafFc-JSpI90Uks5qPxwOgaJpZM4I-nqM>.
>